app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :comments
+  has_many :posts
+  has_many :likes
+
+  def last_posts
+    Post.last(3)
+  end
```

Comment:
- [ ] Please the user model should have a method that returns the 3 most recent posts. See the example below.
```ruby
# app/models/user.rb
...

def recent_post
  posts.order('created_at Desc').limit(3)
end
```